onnx/defs/schema.h

```diff
@@ -463,13 +463,39 @@ class OpSchema {
   };
 };
 
+typedef std::unordered_map<
```

Comment:
It's good to explicitly say what each of these strings mean.

Reply:
I thought the OpName_Domain_Version_Schema_Map name showed its meaning. I can add comments there too.

Reply:
add some comments inline. Thank you!